frontend/src/features/admin-form/create/logic/components/LogicContent/InactiveLogicBlock/FieldLogicBadge.tsx

```diff
@@ -32,7 +32,7 @@ export const FieldLogicBadge = ({ field }: FieldLogicBadgeProps) => {
           <Icon as={fieldMeta.icon} fontSize="1rem" />
         </Tooltip>
         {field.questionNumber ? <Text>{field.questionNumber}.</Text> : null}
-        <Text isTruncated>{field.title}</Text>
+        <Text noOfLines={1}>{field.title}</Text>
```

Comment:
unrelated to this PR: should fix delete box overlapping with long field title, regardless of truncated or not. For reference:

truncated
<img width="698" alt="image" src="https://user-images.githubusercontent.com/25571626/176918885-8c8ec87b-1863-483e-81ec-7c645fffd971.png">

not truncated
<img width="698" alt="image" src="https://user-images.githubusercontent.com/25571626/176919036-0ef0d672-0783-4aa3-a5ca-6761298bb403.png">